siliconcompiler/core.py

```diff
@@ -3366,6 +3368,7 @@ def node(self, flow, step, task, index=0):
            task_module = task
         elif inspect.ismodule(task):
             task_module = task.__name__
+            self.modules[task.__name__] = task
```

Comment:
`self.modules[task_module]` instead of `self.modules[task.__name__]`?

Reply:
(Also, while you're here, it looks like the indentation after the prior `if (isinstance(task, str)):` is 3 spaces instead of 4.)

Reply:
thanks for catching those